packages/codegen-ui-react/lib/__tests__/studio-ui-codegen-react.test.ts

```diff
@@ -44,6 +44,12 @@ describe('amplify render tests', () => {
     it('should generate a simple image component', () => {});
 
     it('should generate a simple string component', () => {});
+
+    it('should generate a simple component without variant specific generation', () => {
+      const generatedCode = generateWithAmplifyRenderer('buttonGolden');
+      expect(generatedCode.componentText.indexOf('restProp')).toEqual(-1);
```

Comment:
I think string.contains would be a better method for tests. could read pretty natural like this I think.

expect(generatedCode.componentText.contains('restProp')).toBe(false);